tagcompare/capture.py

```diff
@@ -175,7 +225,8 @@ def main():
     output.aggregate()
 
     configs = settings.DEFAULT.configs_in_comparisons()
-    __capture_tags_for_configs(cids=cids, pathbuilder=pathbuilder, configs=configs)
+    _capture_tags_for_configs(
+        cids=cids, pathbuilder=pathbuilder, configs=configs)
```

Comment:
Does `return original_build` have any effect here?

Also, if `_capture_tags_for_configs` encounters errors, should the program exit with non-zero exit status? It looks like no, although I'm guessing it should. I would recommend again raising exceptions instead of returning values that indicate an error: it will make this sort of thing much harder to miss.